### PR TITLE
add an implement for calicoctl version --client

### PR DESCRIPTION
**Pull request by cyclinder**
## Description

add an implement for calicoctl version --client

## Related issues/PRs

fixes https://github.com/projectcalico/calico/issues/9244

## Todos

- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
Added support for calicoctl version --client
```

## Reminder for the reviewer

Make sure that this PR has the correct labels and milestone set.

Every PR needs one `docs-*` label.

- `docs-pr-required`: This change requires a change to the documentation that has not been completed yet.
- `docs-completed`: This change has all necessary documentation completed.
- `docs-not-required`: This change has no user-facing impact and requires no docs.

Every PR needs one `release-note-*` label.

- `release-note-required`: This PR has user-facing changes. Most PRs should have this label.
- `release-note-not-required`: This PR has no user-facing changes.

Other optional labels:

- `cherry-pick-candidate`: This PR should be cherry-picked to an earlier release. For bug fixes only.
- `needs-operator-pr`: This PR is related to install and requires a corresponding change to the operator.

### CODE REVIEW

File: calicoctl/calicoctl/commands/version.go [hunk 1]
```diff
@@ -56,6 +56,8 @@ Options:
      --poll=<POLL>             Poll for changes to the cluster information at a frequency specified using POLL duration
                                (e.g. 1s, 10m, 2h etc.). A value of 0 (the default) disables polling.
      --allow-version-mismatch  Allow client and cluster versions mismatch.
+	 --client                  Display the client version only.
```
Comment (caseydavenport) [inline calicoctl/calicoctl/commands/version.go:59]: Indentation looks off here

**Review by caseydavenport — COMMENTED** [review]
One minor comment, but otherwise LGTM - thanks for picking this up @cyclinder !

File: calicoctl/calicoctl/commands/version.go (hunk 1 above)
Comment (cyclinder) [reply calicoctl/calicoctl/commands/version.go:59]: Oh, it looks good in my vscode, let me recheck it.

**Review by caseydavenport — APPROVED** [review]

**Pull request merged**